services/outputhost/cgcache.go

```diff
@@ -444,7 +444,23 @@ func (cgCache *consumerGroupCache) manageConsumerGroupCache() {
 	}
 }
 
-// refreshCgCache contacts metadata to get all the open extents and then loads them if needed
+// refreshCgCacheNoLock is a routine which is called without the extMutex held.
+// It takes the mutex and in turn refreshes the cg by contactingmetadata to
```

Comment:
kind of confused here, don't we hold the lock during the call? Why it's called XXXNoLock()?